src/button.js

```diff
@@ -26,9 +26,14 @@ const Button = (
   let offset, margin, top, height, width, strokeWidth
 
   const color = sx && sx.color ? sx.color : null
```

Comment:
nit: instead of mutating the `sx` object, would it be possible to just isolate the keys we're willing to spread by doing something like:
```suggestion
  const {color, ...sxProp} = sx || {}
```

Reply:
nice, that's cleaner